src/Controls/src/Core/Stepper/Stepper.Mapper.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Microsoft.Maui.Controls;
+public partial class Stepper
+{
+    internal static new void RemapForControls()
+    {
+        StepperHandler.Mapper.AppendToMapping(nameof(Stepper.Increment), Stepper.MapInterval);
```

Comment:
Since this mapper is just to call another mapper, it can be done in shared code (this file):

```cs
    internal static void MapInterval(IStepperHandler handler, IStepper view)
    {
       handler.UpdateValue(nameof(IStepper.Interval));
    }
```

The advantage of just calling the handler update is that if there is some append to the `Interval` mapping, it gets invoked. Using the `handler.PlatformView.UpdateIncrement(view);` method directly means we are creating 2 mappers with the same logic, but the user may no know which to override and thus we may end up with different logic depending on what fires when.

Reply:
Updated the MapInterval method to use handler.UpdateValue(nameof(IStepper.Interval))
 as suggested. Could you please review the changes?